src/main/resources/deployment.conf

```diff
@@ -0,0 +1,45 @@
+akka {
+  actor {
+    provider = "akka.cluster.ClusterActorRefProvider"
+    debug {
+      # enable function of LoggingReceive, which is to log any received message at
+      # DEBUG level
+      # receive = on
+      # lifecycle = on
+    }
+
+  deployment {
+    /lookup/partitionRouter {
+        router = broadcast-group
+        nr-of-instances = 100 //Max number of partitions
+        routees.paths = ["/user/partitionManager"]
+        cluster {
+          enabled = on
+          allow-local-routees = off
+          use-role = lookup
+        }
+      }
+  }
+  }
+
+  remote {
+    log-remote-lifecycle-events = off
+    netty.tcp {
+      # hostname = "127.0.0.1" has to be overwritten
+      port = 2555
+    }
+  }
+
+  cluster {
+    seed-nodes = ["akka.tcp://lookupCluster@master1:2555"]
```

Comment:
At the moment, it works. There's way to overwrite those values with command line arguments and this makes a great default.